tests/test_networks.py

```diff
@@ -1,7 +1,7 @@
 import pytest
 
 from pydantic import AnyUrl, BaseModel, EmailStr, HttpUrl, NameEmail, PostgresDsn, RedisDsn, ValidationError, stricturl
-from pydantic.networks import validate_email
+from pydantic.networks import FileUrl, validate_email
```

Comment:
Better to import it directly from `pydantic`. You need to add `FileUrl` in `__init__`

Reply:
Agreed, done as requested